src/main/java/io/jenkins/infra/repository_permissions_updater/hosting/HostingChecker.java

```diff
@@ -81,7 +81,8 @@ public void checkRequest(int issueID) throws IOException {
             msg.append("\nYou can re-trigger a check by editing your hosting request or by commenting `/hosting re-check`");
         } else {
             msg.append("It looks like you have everything in order for your hosting request. "
-                    + "A human volunteer will check over things that I am not able to check for "
+                    + "A member of the [Jenkins hosting team](https://www.jenkins.io/project/teams/hosting/#members-of-the-hosting-team) "
+                    + "will check over things that I am not able to check for "
```

Comment:
Maybe drop that last "for"
```suggestion
                    + "will check over things that I am not able to check"
```

Reply:
Good catch!